qa/sc_evm_estimateGas.py

```diff
@@ -0,0 +1,205 @@
+            "to": "0xd3CdA913deB6f67967B99D67aCDFa1712C293601",
+            "value": "0x100000000000000000",
+            "gasPrice": "0x4B9ACA00"
+        }
+
+        # result from ethereum mainnet node execution - Internal error: insufficient funds for gas * price + value: address 0x2eebcc96b488045368c94cd327f55ecd012c1e32, have 0, want 1
+        # -32603 = Internal error
+        response = sc_node_1.rpc_eth_estimateGas(request)
+        print(response['error'])
+        assert_equal(-32602, response['error']['code'])
+
+        # Test estimating forging stake
+        # data from AccountTransactionApiRouteTest
+        request = {
+            "from": "0x" + evm_address,
+            "data": "0x5ca748ff1122334455669988112233445566778811223344556677881122334455667788aabbddddeeff0099aabbccddeeff0099aabbccddeeff0099aabbccddeeff00123400000000000000000000000000000000000000000000000000000000000000000000000000000000000000b876fa045fcd3d15a3b6ea51497e06244af17978",
+            "to": "0x0000000000000000000022222222222222222222",
+            "value": "0x1",
+            "gasPrice": "0x4B9ACA00"
+        }
+        response = sc_node_1.rpc_eth_estimateGas(request)
+        assert_equal('0x1c9c380', response['result'])
```

Comment:
`0x1c9c380` equals to 30M - block gas limit.
It cant be like this...
We for sure have a bug.

Reply:
Oh, I did not notice that! I'll directly check why that occurs